components/dropdown/script.js

```diff
@@ -53,6 +53,22 @@ class GamefaceDropdown extends CustomElementValidator {
         this.onMouseLeave = this.onMouseLeave.bind(this);
     }
 
+    get disabled() {
+        return this.hasAttribute('disabled');
+    }
+
+    set disabled(value) {
+        if (value) {
+            this.classList.add('gameface-dropdown-disabled');
+            this.setAttribute('disabled', '');
+            this.setAttribute('tabindex', '-1');
+        } else {
+            this.classList.remove('gameface-dropdown-disabled');
+            this.removeAttribute('disabled');
+            this.setAttribute('tabindex', '1');
```

Comment:
Can you please check if we do the same(add/remove tabindex) for the radio group and the checkbox, because we might've missed this case and please add them if they are missing. 
Thanks.

Reply:
* The radio group (buttons) does set the proper `tabindex` value.
* The Checkbox didn't use `tabindex` -> added.